pkg/conversion/decode.go

```diff
@@ -59,7 +72,9 @@ func (s *Scheme) Decode(data []byte) (interface{}, error) {
 // technique. The object will be converted, if necessary, into the versioned
 // type before being returned. Decode will not decode objects without version
 // set unless version is also "".
-func (s *Scheme) DecodeToVersion(data []byte, version string) (interface{}, error) {
+// a GroupVersion with .IsEmpty() == true is means "use the internal version
+// for this group
```

Comment:
I see, clarify the comment to `a GroupVersion with .IsEmpty() == true is means "use the internal version for the object's group`